science/veryfasttree/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        citius veryfasttree 4.0.3 v
+github.tarball_from archive
+revision            0
+
+categories          science
+license             GPL-3
+platforms           darwin
+maintainers         usc.es:cesaralfredo.pineiro
```

Comment:
the maintainers line is usual `{obfuscated_email @GitHub-handle} openmaintainer`; the latter is to allow others to make minor changes/update to the Portfile without your permission

Reply:
openmaintainer and @GitHub-handle added, done